projects/lib/src/admin/components/page-list/page-list.component.ts

```diff
@@ -98,6 +106,33 @@ export class PageListComponent implements OnInit, OnChanges {
            changes[propertyName].previousValue !== changes[propertyName].currentValue;
   }
 
+  changeSortStrategy(sortBy: string) {
+    if (this.sortBy && this.sortBy === sortBy){
+      sortBy = "!" + this.sortBy;
+      this.sortBy = sortBy;
+    }
+    else {this.sortBy = sortBy}
+    switch (sortBy){
```

Comment:
this can be simplified by removing the switch statement and doing something like this `return this.list.sort((a, b) => (a.Doc[sortBy] > b.Doc[sortBy] ? 1 : -1))` since it looks like the `sortBy` value is the property you need to get from the doc